This bug is the reason why the OCI provisioning fails on the `branch-2026.1`, but not later.
See CI job: https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-14631/5/console
Error from there:
```
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR > Traceback (most recent call last):
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >   File "/home/ubuntu/scylla-cluster-tests/sdcm/tester.py", line 255, in wrapper
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     return method(*args, **kwargs)
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >   File "/home/ubuntu/scylla-cluster-tests/sdcm/utils/decorators.py", line 128, in inner
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     res = func(*args, **kwargs)
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >   File "/home/ubuntu/scylla-cluster-tests/sdcm/tester.py", line 1129, in setUp
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     self._init_params()
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     ~~~~~~~~~~~~~~~~~^^
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >   File "/home/ubuntu/scylla-cluster-tests/sdcm/tester.py", line 829, in _init_params
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     self.params = init_and_verify_sct_config()
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >                   ~~~~~~~~~~~~~~~~~~~~~~~~~~^^
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >   File "/home/ubuntu/scylla-cluster-tests/sdcm/sct_config.py", line 4737, in init_and_verify_sct_config
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     sct_config.verify_configuration()
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >   File "/home/ubuntu/scylla-cluster-tests/sdcm/sct_config.py", line 3957, in verify_configuration
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     self._check_per_backend_required_values(backend)
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >   File "/home/ubuntu/scylla-cluster-tests/sdcm/sct_config.py", line 4183, in _check_per_backend_required_values
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     self._check_backend_defaults(backend, self.backend_required_params[backend])
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >   File "/home/ubuntu/scylla-cluster-tests/sdcm/sct_config.py", line 4190, in _check_backend_defaults
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >     assert _opt["name"] in self, "{} missing from config for {}".format(_opt["name"], backend)
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR >            ^^^^^^^^^^^^^^^^^^^^
20:00:59  < t:2026-05-13 17:00:58,851 f:tester.py       l:260  c:sdcm.tester          p:ERROR > AssertionError: oci_instance_type_db missing from config for oci
```

The error happens in conjunction with the OCI bug where we do not pass through the `oci_instance_type_db` SCT config option down the road to the `runSctTest` pipeline phase and do not run the `provisionResources` one.

--- a/defaults/oci_config.yaml
+++ b/defaults/oci_config.yaml
@@ -9,6 +9,8 @@ instance_provision: "on_demand" # NOTE: DenseIO shapes cannot be 'spot'
 oci_region_name:
   - us-ashburn-1
 
+oci_instance_type_db: "VM.DenseIO.E5.Flex:8"
+
 oci_instance_type_loader: "VM.Standard3.Flex:4:32"
 oci_image_loader: ""  # NOTE: Code will automatically take latest ubuntu:24.04
 ami_loader_user: 'ubuntu'

--- a/jenkins-pipelines/oss/jepsen/jepsen-all.jenkinsfile
+++ b/jenkins-pipelines/oss/jepsen/jepsen-all.jenkinsfile
@@ -4,6 +4,7 @@
 def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 jepsenPipeline(
+    backend: 'gce',
     test_config: 'test-cases/jepsen/jepsen.yaml',
     test_name: 'jepsen_test',
     provision_type: 'on_demand',

--- a/jenkins-pipelines/oss/raft/jepsen-with-raft.jenkinsfile
+++ b/jenkins-pipelines/oss/raft/jepsen-with-raft.jenkinsfile
@@ -4,6 +4,7 @@
 def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 jepsenPipeline(
+    backend: 'gce',
     test_config: 'test-cases/jepsen/jepsen_with_raft.yaml',
     provision_type: 'on_demand',
     post_behavior_db_nodes: 'destroy',

--- a/sdcm/sct_config.py
+++ b/sdcm/sct_config.py
@@ -3317,6 +3317,8 @@ class SCTConfiguration(BaseModel):
                     if cmd.startswith("latte"):
                         script_name_regx = re.compile(r"([/\w-]*\.rn)")
                         script_name = script_name_regx.search(cmd).group(1)
+                        if script_name.startswith("scylla-qa-internal"):
+                            continue
                         full_path = pathlib.Path(get_sct_root_path()) / script_name
                         assert full_path.exists(), f"{full_path} doesn't exists, please check your configuration"
 
@@ -3686,6 +3688,8 @@ class SCTConfiguration(BaseModel):
     def _validate_scylla_d_overrides_files_exists(self):
         if scylla_d_overrides_files := self.get("scylla_d_overrides_files"):
             for config_file_path in scylla_d_overrides_files:
+                if config_file_path.startswith("scylla-qa-internal"):
+                    continue
                 config_file = pathlib.Path(get_sct_root_path()) / config_file_path
                 assert config_file.exists(), f"{config_file} doesn't exists, please check your configuration"
 
@@ -3702,7 +3706,7 @@ class SCTConfiguration(BaseModel):
                 self.backend_required_params["aws"].extend(
                     ["ami_id_vector_store", "instance_type_vector_store", "ami_vector_store_user"]
                 )
-                self._check_backend_defaults(backend, self.backend_required_params[backend])
+            self._check_backend_defaults(backend, self.backend_required_params[backend])
         else:
             raise ValueError("Unsupported backend [{}]".format(backend))
 

